WooCommerce/Classes/ViewRelated/Products/Cells/Product Images/Collection View Cells/ExtendedAddProductImageCollectionViewCell.swift

```diff
@@ -0,0 +1,46 @@
+import UIKit
+
+class ExtendedAddProductImageCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var title: UILabel!
```

Comment:
ah I see that these are not private so that the subviews can be tested. Generally, I prefer testing the view model to the subviews so that the views don't have to be exposed just for testing. But it seems like we have previous examples for the cell subview tests, so this looks good and please ignore the comment above 👍